mhs/outbound/mhsapi.json

```diff
@@ -1 +1 @@
tion ID when producing it's logs, then that should be passed in here, so that logs for a single request can be tied together across the supplier system and the MHS.\n\nWhen performing async requests where the response is put on the inbound queue, the correlation ID will be put with the response on the queue.\n\nNote that this correlation ID gets sent to/from Spine."}, {"name": "ods-code", "in": "header", "required": false, "schema": {"type": "string"}, "description": "ODS Code receiving system. It defaults to Spines ODS Code if not porvided and is primarily used for indirect messaging, i.e. forward reliable for example, where the destination system is not Spine. The ODS Code is used to lookup the constract properties in SDS."}], "responses": {"200": {"description": "Successful response from Spine.", "content": {"text/xml": {}}}, "202": {"description": "Acknowledgement that we successfully sent the message to Spine (response will come asynchronously on the inbound queue)."}}, "requestBody": {"required": true, "content": {"application/json": {"schema": {"$ref": "#/definitions/RequestBody"}}}, "description": "The HL7 payload (and optional attachments) to be sent to Spine."}}}, "/healthcheck": {"get": {"summary": "Healthcheck endpoint", "description": "This endpoint just returns a HTTP 200 response and does no further processing. This endpoint is intended to be used by load balancers/other infrastructure to check that the server is running.", "operationId": "getHealthcheck", "responses": {"200": {"description": "The only response this endpoint returns."}}}}}, "info": {"title": "MHS API docs", "version": "0.1"}, "openapi": "3.0.2", "components": {"schemas": {"Attachment": {"type": "object", "properties": {"is_base64": {"type": "boolean", "description": "Whether the attachment payload is base64-encoded or not. This is only required for binary attachments eg images."}, "payload": {"type": "string", "minLength": 1, "maxLength": 5000000, "description": "The attachment, possibly base64-encoded as per is_base64."}, "description": {"type": "string", "minLength": 1, "maxLength": 100, "description": "Description of the attachment"}, "content_type": {"type": "string", "enum": ["text/plain", "text/html", "application/pdf", "text/xml", "application/xml", "text/rtf", "audio/basic", "audio/mpeg", "image/png", "image/gif", "image/jpeg", "image/tiff", "video/mpeg", "application/msword", "application/octet-stream"], "description": "Content type of the attachment"}}, "required": ["content_type", "description", "is_base64", "payload"]}, "RequestBody": {"type": "object", "properties": {"attachments": {"default": [], "maxItems": 98, "description": "Optional attachments to send with the payload. Only for use for interactions that support sending attachments.", "type": "array", "items": {"$ref": "#/components/schemas/Attachment"}}, "payload": {"type": "string", "minLength": 1, "maxLength": 5000000, "description": "HL7 Payload to send to Spine"}}, "required": ["payload"]}}}}
```

Comment:
does it work with pretty printed JSON?
might be easier to review in the future.

Reply:
it was auto-generated by `pipenv run generate-openapi-docs`

Reply:
no switch in the generate-openapi-docs api?

Reply:
it happens that's quite easy to add pretty print - so as requested I've added it